pkg/trace/obfuscate/obfuscate.go

```diff
@@ -44,6 +45,25 @@ type Config struct {
 
 	// Redis enables obfuscatiion of the "memcached.command" tag for spans of type "memcached".
 	Memcached bool
+
+	// Set to 1 if enabled 0 if not. We're using an integer
+	// so we can use the sync/atomic for thread-safe access.
+	ignoreEscapes int32
+}
+
+// SetIgnoreEscapes sets whether or not escape characters should be ignored,
+// i.e. treated as literal characters.
+func (o *Obfuscator) SetIgnoreEscapes(val bool) {
```

Comment:
This would then become `SetLiteralEscapes(ok bool)`. I don't feel strongly about the name change, but the argument name should be `ok` or `yes`. I think that's clearer.